maintainers/maintainer-list.nix

```diff
@@ -333,6 +333,11 @@
     github = "andersk";
     name = "Anders Kaseorg";
   };
+  anderslundstedt = {
```

Comment:
Please add this as an initial separate commit.

Reply:
Done.